admin/class-acc_user_importer-admin.php

```diff
@@ -448,7 +463,7 @@ private function proccess_user_data ( $users ) {
 
 			//--------CREATE NEW USER-----
 			$this->log_dual(" > email not found on any other users");
-			$new_users[] = $userContactId;
+			$new_users[] = $loginName;
```

Comment:
This is used for logging, and I meant this to display the peoples name.  Here is an example. 
--Created account for 1 people:
  274274 (reneedd2@hotmail.com)
  
  So let`s change the above line for something like line 428
  $new_users[] = $accUserData['display_name'];